### ISSUE TITLE
Anthropogenic and biogenic VOC in InMAP

### ISSUE THREAD

**Issue by bkoo-git** [BAAQMD/InMAP-SFAB#10]
Based on variables in the InMAP chemistry input data, it appears that InMAP distinguishes anthropogenic and biogenic VOCs (aVOC vs. bVOC), and gas/particle partitioning of organics is handled separately for anthropogenic and biogenic SOAs (aOrgPartitioning vs. bOrgPartitioning).
According to the InMAP documentation, however, the acceptable species name for VOC emissions is just 'VOC'.
My questions are, then:
1. How can one tell InMAP whether the 'VOC' emissions in an emissions input file are anthropogenic or biogenic?
2. If no such info is given, will InMAP automatically assume anthropogenic VOC emissions and use 'aOrgPartitioning' to produce SOA from the VOC emissions?

I'm including @ctessum @yuzhou-wang @bujinb @pmartien @stephenreid65 @dholstius in the discussion.

**Comment by ctessum** [comment]
InMAP just automatically assumes that VOC emissions are anthropogenic. As you have seen, there is some code and data in there to separately treat anthropogenic and biogenic VOCs, but we've never gotten around to actually using it.

**Comment by bkoo-git** [comment]
Thanks @ctessum for the quick clarification!
The 2017 PLOS paper (https://doi.org/10.1371/journal.pone.0176131) states, "For partitioning between VOCs and secondary organic aerosol (SOA) we only consider those VOCs that are SOA precursors". I believe this should apply to the VOC emissions, too; i.e., you should include only the SOA precursor VOCs when preparing your VOC emissions input, not the whole list of VOC species, right?

**Comment by ctessum** [comment]
Yes, theoretically only SOA precursor VOCs should be included. Owing mainly to a mistake in our emissions processing, results that we've published so far have included all non-methane VOCs instead of only the SOA precursors. Some colleagues at Carnegie-Mellon have recently looked into this in the process of comparing InMAP to another model, and I think they found that it makes some difference, but not a huge difference.
